cypress/page/login.page.ts

```diff
@@ -0,0 +1,26 @@
+class LoginPage{
+    private email : string;
+    private password : string;
+    private emailSelector : string;
+    private passwordSelector : string;
+    private submitSelector : string;
+
+    constructor(){
+        this.email = "aperdomobo@gmail.com";
+        this.password = "WorkshopProtractor";
```

Comment:
This is what we call hard code, because this page object will only help to login just one user, if these properties become input parameters of the login method it can login any user 😉